common-app-lib/src/cmr/common_app/config.clj

```diff
@@ -17,3 +17,7 @@
   "Flag for whether or not launchpad token is enforeced."
   {:default false
    :type Boolean})
+
+(defconfig release-version
+  "Contains the release version of CMR."
+  {:default "1.1.1-r1.1.1"})
```

Comment:
I noticed that Legacy services uses a default version of "dev". I think dev is better to indicate it's not a specific version - this numbering makes it look like a real release.

Reply:
done.